jenkins/e2e-image/e2e-runner.sh

```diff
@@ -426,8 +426,8 @@ if [[ "${USE_KUBEMARK:-}" == "true" ]]; then
 fi
 
 e2e_go="$(dirname "${0}")/e2e.go"
-if [[ ! -f "${e2e_go}" ]]; then
-  echo "TODO(fejta): stop using head version of e2e.go"
+if [[ ! -f "${e2e_go}" || -e "./hack/jenkins/.use_head_e2e" ]]; then
```

Comment:
Hmm? I don't want it to rely on the test-infra repo because I just want to set it in a kubernetes PR and then unset before checking in. We use this pattern in a few places.

It might be a good idea to add a simple verify check that these don't get checked in, actually.